tests/Agent/IntegrationTests/Applications/AzureFunctionApplication/AzureFunctionApplication.csproj

```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net6.0;net8.0</TargetFrameworks>
+    <TargetFrameworks>net8.0</TargetFrameworks>
```

Comment:
Still need to target 2 versions - 8 and 9

Reply:
I was having problems building this project for net9.0 but they have been resolved.